client-tools/Export/ExportFeature.cpp

```diff
@@ -98,11 +98,23 @@ void ExportFeature::collectOptions(std::shared_ptr<options::ProgramOptions> opti
       "restrict to collection name (can be specified multiple times)",
       new VectorParameter<StringParameter>(&_collections));
 
-  options->addOption("--query", "AQL query to run", new StringParameter(&_query));
-  
-  options->addOption("--query-max-runtime", "runtime threshold for AQL queries (in seconds, 0 = no limit)", 
-                     new DoubleParameter(&_queryMaxRuntime))
-                     .setIntroducedIn(30800);
+  options->addOldOption("--query", "custom-query");
+  options
+      ->addOption("--custom-query", "AQL query to run", new StringParameter(&_customQuery))
+      .setIntroducedIn(31000);
+  options->addOldOption("--query-max-runtime", "custom-query-max-runtime");
+  options
+      ->addOption(
+          "--custom-query-max-runtime",
+          "runtime threshold for AQL queries (in seconds, 0 = no limit)",
+          new DoubleParameter(&_customQueryMaxRuntime))
+      .setIntroducedIn(31000);
```

Comment:
I am undecided whether we should the change the version number here. It is true that the option didn't exist before under this name, but it existed before under the old name. Not sure what our convention for this is...
@Simran-B @ansoboleva : any idea?

Reply:
@jsteemann Since the options existed before and this here is only a name change, I would lean towards keeping the version number as it was. Otherwise, it might give a false assumption.

Reply:
@ansoboleva : fine for me.
@cpjulia : can you please change the version number back?

Reply:
Addressed